ruff.toml

```diff
@@ -0,0 +1,25 @@
+select = [
+    "B",   # flake8-bugbear
+    "C4",  # flake8-comprehensions
+    "COM", # flake8-commas
+    "E",   # pycodestyle Error
+    "F",   # Pyflakes
+    "I",   # isort
+    "PERF",# Perflint
+    "PIE", # flake8-pie
+    "PT",  # flake8-pytest-style, would be nice to enable
+    "RET", # flake8-return
+    "RUF", # Ruff-specific rules
+    "SIM", # flake8-simplify
+    "UP",  # pyupgrade
+    "W",   # pycodestyle Warning
+]
+ignore = [
+    "B017",    # assert-raises-exception
+    "B022",    # useless-contextlib-suppress
+    "RUF002",  # you shall not touch my guillemets
+    "RUF003"   # Comment contains ambiguous character
```

Comment:
Have you checked whether we still need those that you copied from packit?
Maybe we don't actually need to ignore them here in ogr.

Reply:
I'll have a look at it, there is still one issue with “dissecting” the exceptions in tests :/